request.go

```diff
@@ -147,6 +147,20 @@ func (r *Req) SetForm(files []map[string]string, fields []map[string]string) *Re
 	return r
 }
 
+// SetProxy sets proxy URL to http client
+func (r *Req) SetProxy(u string) *Req {
+	proxyURL, err := url.Parse(u)
+	if err != nil {
+		r.err = err
+		return r
+	}
+	transport := http.Transport{}
```

Comment:
Please do not overwrite all transport layer. There are different configurations there. Just update default r.client.Transport's proxy setting. Something like:
`r.client.Transport.(*http.Transport).Proxy = http.ProxyURL(proxyURL)`

Reply:
I fixed in https://github.com/binalyze/httpreq/pull/9/commits/910620d4f1125244f0e4c05b59298be23b9f45d8.